python/cuml/dask/common/base.py

```diff
@@ -17,6 +17,12 @@
 import cupy as cp
 import dask
 import numpy as np
+import warnings
+from toolz import first
+from collections.abc import Iterable
+from sklearn.exceptions import NotFittedError
```

Comment:
Do we have sklearn as a standard requirement? At some point it was only needed for development. (It's not in meta.yaml)